README.adoc

```diff
@@ -39,19 +45,17 @@ https://github.com/keep-network/keep-core/issues[is appreciated!]
 dApp developers will be most interested in the smart contracts exposing Keep's
 on-chain facilities.
 
-The contracts can be found in the link:solidity/[`solidity/`] directory. They
-can be used to request miner-resistant random numbers, as well as handle
-creating and managing keeps, generating new secrets, and requesting signatures.
-
-It runs the Keep network’s random beacon and participates in related
-computations.
+The core contracts can be found in the link:solidity/[`solidity/`] directory.
+They can be used to request
+link:solidity/contracts/IRandomBeacon.sol[miner-resistant random numbers], as
+well as creating and managing keeps. To generate new ECDSA key material and
+request signatures, the contracts can be found in
+link:https://github.com/keep-network/keep-ecdsa/blob/master/solidity/contracts/api/IBondedECDSAKeep.sol[`keep-ecdsa`].
```

Comment:
Hmmm… I feel like you're either linking to the contracts, so solidity dir in keep-ecdsa, or to the interface for Bonded ECDSA keeps, in which case it's worth saying as much.

Really I'm not convinced leaving the interface names out yet linking directly to the interfaces is a great idea. Not a blocking note, just something to think about.

Reply:
Including the interface name in an introductory README seems like overkill. In my mind this doc is about convincing people to dig deeper + acting as an index of our work, rather than familiarizing people with specific code details immediately.

That said, if we have a killer code snippet for dApp devs that'd be a nice alternative to either

Reply:
Will give it some more thought :thinking:

Reply:
I guess what I really want is a link to a

* "how to use the random beacon" doc, or alternatively "random beacon Solidity API docs"
* "how to open a keep and do something with it" doc, or alternatively "all ECDSA keep Solidity API docs"

Reply:
^^ agreed. For the next pass :)